src/environmentd/tests/sql.rs

```diff
@@ -349,23 +346,13 @@ fn test_tail_negative_diffs() -> Result<(), Box<dyn Error>> {
 fn test_tail_basic() -> Result<(), Box<dyn Error>> {
     mz_ore::test::init_logging();
 
-    // Set the timestamp to zero for deterministic initial timestamps.
-    let nowfn = Arc::new(Mutex::new(NOW_ZERO.clone()));
-    let now = {
-        let nowfn = Arc::clone(&nowfn);
-        NowFn::from(move || (nowfn.lock().unwrap())())
-    };
-    let config = util::Config::default().workers(2).with_now(now);
```

Comment:
It was unclear to me why it was needed, and I thought that it might be making the test a bit complicated than it needed to be. I can add it back though if you disagree.

Reply:
The comment here doesn't explain why we need deterministic timestamps, so if things still pass without it I'm fine.